app/core/constants.ts

```diff
@@ -0,0 +1,54 @@
+﻿namespace dogsrus.virtdog {
+  export class EventNames {
```

Comment:
I would do the same thing for the constants that I recommended for the config value above, except I would wrap the object in Object.freeze to prevent modification.